frontend/app/src/components/status/FrontendUpdateNotifier.vue

```diff
@@ -12,42 +12,11 @@
 </template>
 
 <script lang="ts">
-import { Component, Vue } from 'vue-property-decorator';
+import { Component, Mixins } from 'vue-property-decorator';
+import RegisterSwMixin from '@/mixins/register-sw-mixin';
```

Comment:
Rather than introducing a mixin shouldn't we just migrate the component to composition?